docs/developers/hacking.rst

```diff
@@ -188,6 +188,156 @@ virtual environment (using the :command:`workon` command) and running the
+
+Pootle Set Up
+^^^^^^^^^^^^^
+
+.. note:: For convenient these instructions consistently specify paths ``C:\venv`` and ``C:\git\pootle``, but you can change these to suit your environment and needs.
+
+For installing the dependencies in an isolated environment, set up a fresh virtualenv.
+
+.. code-block:: console
+
+    > python -m pip install virtualenv
+    > virtualenv C:\venv
+
+Activate the new virtualenv and upgrade pip:
+
+.. code-block:: console
+
+    > C:\venv\Scripts\activate
+    (venv)> python -m pip install --upgrade pip
+
+Go to the pootle ``requirements\base.txt`` and comment out the following packages:
+
+.. code-block:: console
+
+    # lxml
+    # levenshtein
+    # scandir
+
+These three packages are difficult to build on Windows, so we will download pre-built versions to install manually 
+further on.
+
+Also change any ``~=`` compatible-version markers into ``==`` exact-version markers, 
+as Windows-based ``pip`` doesn't seem to like these.
+
+To avoid permissions issues, required packages have to be downloaded and installed in separate steps. 
+By default, pip stores temporary files in your ``user\AppData`` folder which may not allow access in 
+all circumstances. There are also some packages that need to be downloaded and installed manually.
+Create a temporary folder somewhere that any user can access without permission issues (e.g. ``C:\temp``).
+If you are seeing "directory was not empty" or "file not found" errors during a ``pip install``, then these 
+next commands will solve the problem.
+
+.. code-block:: console
+
+    (venv)> python -m pip install -r requirements\dev.txt -b C:\temp -d C:\temp
+    (venv)> python -m pip install -r requirements\dev.txt -b C:\temp -t C:\venv\Lib\site-packages\ --no-index --find-links="C:\temp"
+
+Next, download the appropriate installers for your system and Python version for the special requirements, 
+saving them into your temporary folder.
+
+- http://www.lfd.uci.edu/~gohlke/pythonlibs/#lxml
+- http://www.lfd.uci.edu/~gohlke/pythonlibs/#Python-Levenshtein
+- http://www.lfd.uci.edu/~gohlke/pythonlibs/#scandir
+
+Now install them explicitly:
+
+.. code-block:: console
+
+    (venv)> python -m pip install C:\temp\lxml-3.6.4-cp27-cp27m-win32.whl
+    (venv)> python -m pip install C:\temp\python_Levenshtein-0.12.0-cp27-none-win32.whl
+    (venv)> python -m pip install C:\temp\scandir-1.2-cp27-none-win32.whl
+
+Now that all the requirements are lined up, we are ready to initialise Pootle.
+
+Clone your fork of the Pootle master using your favourite Windows implementation of Git so that you have a working 
+copy somewhere accessible on your computer (e.g. ``C:\git\pootle``).
+
+First, rename ``90-dev-local.conf.sample`` to ``90-dev-local.conf`` (in ``pootle\settings``) to enable a basic 
+configuration suitable for local hacking. Then initialise your local Pootle working copy into the virtualenv:
+
+.. code-block:: console
+
+    (venv)> cd C:\git\pootle
+    (venv)> pip install -e .
```

Comment:
Note we've moved off the separate `-r` step on the Linux install docs.  Not sure if that would work on Windows because of some of the build issues you hit in previous steps.

But it would make sense to do:

`pip install -e .[dev]`

That will install all dev requirements.